docs/release/trg-0/trg-4-08.md

```diff
@@ -0,0 +1,49 @@
+---
+title: TRG 4.08 - Multi-platform
```

Comment:
```suggestion
title: TRG 4.07 - Multi-platform images
```
i guess next number for a TRG within the container section would be 4.07

Reply:
sorry missed that ther is already a 4.07

Reply:
There is a TRG 4.07 in TRG 0 section, so 4.08 shuld be fine for this proposal.